mysql-test/suite/galera/t/galera_log_bin.test

```diff
@@ -50,5 +50,102 @@ SHOW BINLOG EVENTS IN 'mysqld-bin.000001' FROM 123;
 DROP TABLE t1;
 DROP TABLE t2;
 
+--echo #
+--echo # Test that disabling binlog does not prevent Galera replication,
+--echo # but events are not recorded in the binlog
+--echo #
+--connection node_2
+RESET MASTER;
+
+--connection node_1
+RESET MASTER;
+
+SET SESSION sql_log_bin = OFF;
+CREATE TABLE t1 (a int primary key);
+INSERT INTO t1 VALUES (1);
+BEGIN;
+INSERT INTO t1 VALUES (10);
+INSERT INTO t1 VALUES (11);
+INSERT INTO t1 VALUES (12);
+COMMIT;
```

Comment:
Can we add an assert that nothing is logged into the binlog?
```
--let $master_pos_before= query_get_value(SHOW MASTER STATUS,Position,1)
SET SESSION sql_log_bin = OFF;
CREATE TABLE t1 (a int primary key);
INSERT INTO t1 VALUES (1);
BEGIN;
INSERT INTO t1 VALUES (10);
INSERT INTO t1 VALUES (11);
INSERT INTO t1 VALUES (12);
COMMIT;

# Assert that the above statements are not written to binlog.
--let $master_pos_after= query_get_value(SHOW MASTER STATUS,Position,1)
--let $assert_text= assert that the above events are not written to binlog
--let $assert_cond= $master_pos_before = $master_pos_after
--source include/assert.inc
```

Reply:
done